core-dev/packages/coqide/coqide.dev/opam

```diff
@@ -7,8 +7,6 @@ dev-repo: "git+https://github.com/coq/coq.git"
 license: "LGPL-2.1"
 
 depends: [
-  "ocaml" {>= "4.02.3"}
```

Comment:
I can understand removing `camlp5`, but what's the intuition for removing `ocaml` as a dependency, when compilation is required? Same for the `8.10.dev` package.

Reply:
This depends on Coq, which already fixes the ocaml version we need. Seems redundant to have that here *again*.

CoqIDE and Coq are tightly coupled as packages, so this seems fine to me. But I can also revert this part if you prefer.

Reply:
@gares can you comment here? I think this is a policy decision where either the archive should either have `ocaml` _everywhere_ or only exactly where needed (and the latter is hard to determine). Same issue comes up in #687.